package.json

```diff
@@ -21,12 +21,16 @@
     "@next/mdx": "^12.3.0",
     "focus-visible": "^5.2.0",
     "framer-motion": "^7.3.2",
+    "gray-matter": "^4.0.3",
+    "js-yaml": "^4.1.0",
     "next": "^13.0.2",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-markdown": "^8.0.3",
```

Comment:
do we need `react-markdown` as a dep if we're using MDX?

Reply:
This was added in https://github.com/ethereum/geth-website/pull/28, I hadnt merged master back in yet. Used that as a base branch for this